gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -30,10 +30,18 @@
           </div>
           <div class="form-group" ng-class="{'has-error': folderEditor.folderPath.$invalid && folderEditor.folderPath.$dirty && !editingFolderDefaults()}">
             <label translate for="folderPath">Folder Path</label>
-            <input name="folderPath" ng-readonly="editingFolderExisting()" id="folderPath" class="form-control" type="text" ng-model="currentFolder.path" list="directory-list" ng-required="!editingFolderDefaults()" ng-aria-required="!editingFolderDefaults()" path-is-sub-dir />
-            <datalist id="directory-list">
-              <option ng-repeat="directory in directoryList" value="{{ directory }}" />
-            </datalist>
+            <div class="input-group">
+              <input name="folderPath" ng-readonly="editingFolderExisting()" id="folderPath" class="form-control" type="text" ng-model="currentFolder.path" list="directory-list" ng-required="!editingFolderDefaults()" ng-aria-required="!editingFolderDefaults()" path-is-sub-dir />
+              <datalist id="directory-list">
+                <option ng-repeat="directory in directoryList" value="{{ directory }}" />
+              </datalist>
+              <span class="input-group-btn">
+                <button class="btn btn-default" type="button" ng-click="openFolderPicker()" ng-disabled="editingFolderExisting()">
+                  <i class="fas fa-folder-open"></i>
```

Comment:
```suggestion
                  <span class="fa fa-folder-open"></span>
```

Please change all `<i>` tags used for icons to `<span>` and also use just `fa` instead of `fas` (as the latter syntax with `fas` is outdated).